recipes/effcee/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class EffceeConan(ConanFile):
+    name = "effcee"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/effcee/"
+    description = "Zstandard - Fast real-time compression algorithm"
+    topics = ("conan", "effcee", "strings", "algorithm", "matcher")
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package_multi"
+    settings = "cppstd", "os", "arch", "compiler", "build_type"
```

Comment:
Its complaining about this too

```suggestion
    settings = "os", "arch", "compiler", "build_type"
```